lib/helpers.bash

```diff
@@ -230,7 +230,12 @@ _disable-thing ()
         do
             plugin=$(basename $f)
             if [ -e $BASH_IT/$subdirectory/enabled/$plugin ]; then
-                rm $BASH_IT/$subdirectory/enabled/$(basename $plugin)
+                cmd_output="$(rm -f $BASH_IT/$subdirectory/enabled/$(basename $plugin) 2>&1)"
+                if [[ $? -ne 0 ]]; then
```

Comment:
Since these 5 lines are pretty much repeated several times, can we try to move them into a function?